src/olympia/abuse/tasks.py

```diff
@@ -172,6 +178,11 @@ def sync_policies(policies, parent_id=None):
         response.raise_for_status()
         data = response.json()
         sync_policies(data)
+        CinderPolicy.objects.exclude(
+            Q(cinderdecision__id__gte=0)
+            | Q(reviewactionreason__id__gte=0)
+            | Q(modified__date__gte=datetime.today())
```

Comment:
Instead of using `datetime.today()` you should put the current `datetime.now()` in a variable before processing the sync. Otherwise, there is a possible race condition if things start syncing before midnight UTC but you reach that line right after... that would delete what we just synced...

Reply:
https://github.com/mozilla/addons-server/pull/22255/commits/2ab22283fb40d1c16623269402638dfa8c338137